source/develop/release-management/features/storage/multipath-events.html.md

```diff
@@ -0,0 +1,174 @@
+---
+title: Multipath Events
+category: feature
+authors: Fred Rolland
+feature_name: Multipath Events
+feature_modules: engine,vdsm
+feature_status: Design
+---
+
+# Multipath Events
+
+
+## Overview
+
+This feature adds the ability to display events to the user in
+the engine UI when multipath device path changes its status.
+The supported events are when a path moves to down or up state.
+With these events, the user will be able to detect storage issues faster.
```

Comment:
I'm not convinced that logging raw events is useful, and it may also be harmful, hiding other audit logs with 1000's of unhelpful detailed logs about single paths.